1.0/docs/devguide/styling.md

```diff
@@ -8,11 +8,27 @@ subtitle: Developer guide
 
 {% include toc.html %}
 
-
 Polymer uses [Shadow DOM styling
 rules](http://www.html5rocks.com/en/tutorials/webcomponents/shadowdom-201/) for
-providing scoped styling of the element's local DOM.  Scoped styles should be
-provided via `<style>` tags placed inside the element's local DOM `<template>`.
+providing scoped styling of the element's local DOM. This allows elements to
+create style rules that will only apply to the element's light DOM or local
+DOM, as well as advanced theming with CSS custom properties and CSS mixins.
+
+Scoped styles should be provided via `<style>` tags placed inside the element's
+local DOM `<template>`.
+
+To place styles outside of the element, or share styles between elements, you
```

Comment:
This line is not quite right. Style modules are for sharing styles, yes. The `custom-style` element is what you use for defining local DOM aware style rules outside of an element.